dev-tools/deploy

```diff
@@ -12,17 +11,14 @@ def main():
     args = parser.parse_args()
 
     dir = os.path.dirname(os.path.realpath(__file__))
-    os.chdir(dir + "/packer")
+    os.chdir(dir + "/../")
     print("Getting dependencies")
-    check_call("make deps", shell=True)
     check_call("make clean", shell=True)
```

Comment:
Hm, looking at `dev-tools/packer/scripts/Makefile` there is also a clean that should probably be called here?